base/cvd/MODULE.bazel

```diff
@@ -82,3 +82,12 @@ git_repository(
     commit = "952f776f6573aafbb62ea717d871cd1d6816c387", # v1.4.309
     remote = "https://github.com/KhronosGroup/Vulkan-Headers.git",
 )
+
+git_repository(
```

Comment:
FYI on the internal side we'll need to do a go/thirdparty import for this library.

Reply:
yes indeed, was looking into it and there's no libcn-cbor.